src/instance.js

```diff
@@ -480,7 +480,7 @@ Instance.prototype.createTable = function(name, options, callback) {
     },
     function(...args) {
       if (args[1]) {
-        var table = self.table(args[1].name);
+        var table = self.table(args[1].id);
```

Comment:
`args` is a list of `Table` proto objects.  Table only has [`name`](https://github.com/googleapis/googleapis/blob/master/google/bigtable/admin/v2/table.proto#L99), not `id`.  I think that you need to use `args[1].name.split('/').pop()`